docker-services.yml

```diff
@@ -62,11 +62,12 @@ services:
       - "15672:15672"
       - "5672:5672"
   es:
-    image: docker.elastic.co/elasticsearch/elasticsearch-oss:6.2.2
+    image: docker.elastic.co/elasticsearch/elasticsearch-oss:7.3.0
     restart: "always"
     environment:
       - bootstrap.memory_lock=true
       - "ES_JAVA_OPTS=-Xms512m -Xmx512m"
+      - discovery.type=single-node
```

Comment:
Just out of curiosity, is this just a test setting or it will be deployed in single node clusters?

Reply:
Production will be a cluster. I guess this was added by @equadon for the dev. version